.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+jobs: 
+  build: 
+    docker:
+      # TODO: update the image to be stable
+      # CircleCI Go images available at: https://hub.docker.com/r/circleci/golang/
+      - image: circleci/golang
+
+        # directory where steps are run. Path must conform to the Go Workspace requirements
+    working_directory: /go/src/github.com/kubic-project/dex-operator
+
+    environment: # environment variables for the build itself
+      TEST_RESULTS: /tmp/test-results # path to where test results will be saved
+
+    steps: # steps that comprise the `build` job
+      - checkout # check out source code to working directory
+      - run: mkdir -p $TEST_RESULTS # create the test results directory
+   
+      - run: | 
+              make
+
+      - run:
+          name: Check golang style with gofmt, go vet and golint
+          command: make check 
+
+      - run:
+          name: Run unit-tests
+          command: |
+            make test 
```

Comment:
Minor, can we follow the same style in all runs?

Reply:
yop

Reply:
i think the only diff is the `name`. to me this is ok, i will fix on ther projects.

From experience i would remove any additional variable in a template project/file.

Where the var  here is the string name in the `Run unit-tests in XX` i think is better to have just `Run unit-tests` because if we have another new operator project, we will copy and paste and having the wrong name. Is like copyright years and so on . :smile: